lisa/testsuite.py

```diff
@@ -640,6 +640,13 @@ def start(
                 log=case_log,
             )
 
+            if environment.platform:
```

Comment:
1. Never call private method out of the class. It breaks the design and fail other platform.
2. In test cases, use node.features.is_supported to check if SerialConsole is available, and then call it. Don't call platform directly.